pkg/tfvars/gcp/gcp.go

```diff
@@ -25,14 +33,15 @@ type config struct {
 }
 
 // TFVars generates gcp-specific Terraform variables launching the cluster.
-func TFVars(auth Auth, masterConfigs []*gcpprovider.GCPMachineProviderSpec, imageURI, publicZoneName string) ([]byte, error) {
+func TFVars(auth Auth, network Network, masterConfigs []*gcpprovider.GCPMachineProviderSpec, imageURI, publicZoneName string) ([]byte, error) {
```

Comment:
isn't the network information already available in the master and worker machine objects?